File: app/src/main/java/com/example/coffeehelper/MainActivity.java
package com.example.coffeehelper;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.view.Window;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        this.requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.activity_main);

        Button btnGo = (Button) findViewById(R.id.go_button);

        btnGo.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent mine = new Intent(v.getContext(), BrewSelectorActivitiy.class);
                startActivityForResult(mine, 0);
            }
        });
    }


}

File: app/src/main/java/com/example/coffeehelper/MainActivity.kt
package com.example.coffeehelper

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.view.Window
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.activity_main)
        val btnGo = findViewById<View>(R.id.go_button) as Button
        btnGo.setOnClickListener { v ->
            val mine = Intent(v.context, BrewSelectorActivitiy::class.java)
            startActivityForResult(mine, 0)
        }
    }
}